lib/ui/pages/login_page.dart

```diff
@@ -0,0 +1,124 @@
+part of 'pages.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  TapGestureRecognizer _signupButtonTapRecognizer = TapGestureRecognizer();
```

Comment:
There are a lot of suggestions regarding the code that we can see in the image below for the best practice. For example 
```suggestion
 final TapGestureRecognizer _signupButtonTapRecognizer = TapGestureRecognizer();
```
this private variable could be final

for other suggestions, we can see in the image below
<img width="797" alt="Screen Shot 2022-12-05 at 16 40 15" src="https://user-images.githubusercontent.com/44092368/205604793-f9f1209a-0671-49b9-8478-372ce8a699e9.png">